### ISSUE TITLE
Add stop names autocomplete API endpoint

### ISSUE THREAD

**Issue by tomekzaw** [wietlabs/krk_meetings#126]
Create API endpoint (e.g. `/api/v1/stops/autocomplete?query=agh`) that returns a list of suggested stop names in the following format: 
```json
{
  "stop_names": [
    "AGH / UR",
    "Miasteczko Studenckie AGH"
  ]
}
```
Functional requirements:
* The pattern should consist of at least 3 alphanumeric characters.
* The list of matched stop names should be sorted lexicographically.
* The number of results should be limited to 10.

Optionally:
* Stop names abbreviations should also be handled, e.g. match `rmat` to "Rondo Matecznego" and `cmpr` to "Czerwone Maki P+R".

**Comment by tomekzaw** [comment]
In fact, let's return list of all stops with coordinates at `/api/v1/stops`:
```json
{
  "stops": [
    {
      "name": "AGH / UR",
      "latitude": 50.01234,
      "longitude": 19.95912,
    }
  ]
}

**Comment by tomekzaw** [comment]
Closed in #151
